pack.ps1

```diff
@@ -67,6 +71,10 @@ function createNuGetMetaPackage()
 
     echo "Project file in '$csprojFilePath'"
 
+    $icon = [IO.Path]::Combine($env:GITHUB_WORKSPACE, $artifactsFolderName, "icon.png")
```

Comment:
Could be moved into a function

Reply:
I don't think to create a reusable function to a one liner function call

Reply:
I faced an issue to selected the added lines

Reply:
It’s just copying file from a PathA to PathB, PathB is a Package folder path

not sure why it’s an issue?

Reply:
Not an issue, but it would be nice to move them into a single place that will help in case if we change anything, no need to change in two places

Reply:
It’s already inside a function and each function has it own destination folder defined via variable, 

I don’t see any reason to create another function to just to call copy item

Reply:
Maybe just have `$env:GITHUB_WORKSPACE/.github/workflows/nuget-icon.png` as `$iconFilePath` to reuse the value and follow the same pattern as the code above

Reply:
moved to variable